delta-lake/common/src/main/scala/com/nvidia/spark/rapids/delta/DeltaProviderImplBase.scala

```diff
@@ -32,8 +32,8 @@ abstract class DeltaProviderImplBase extends DeltaProvider {
       ),
       GpuOverrides.exec[RapidsDeltaWriteExec](
         "GPU write into a Delta Lake table",
-        ExecChecks.hiddenHack(),
-        (wrapped, conf, p, r) => new RapidsDeltaWriteExecMeta(wrapped, conf, p, r)).invisible()
+        ExecChecks(TypeSig.all, TypeSig.all),
```

Comment:
So we support all data types for writes to delta? Why is this a part of a read deltalake change? Or are we doing to do both here?